frontend/src/store/modules/place/actions.js

```diff
@@ -27,13 +26,42 @@ export default {
                 .then( (response) => {
                     const currentPlace = response.data.data;
                     context.commit('SET_CURRENT_PLACE', currentPlace);
-                    let transformedCurrentPlaceReviews = normalizerService.normalize({
-                        data: currentPlace.reviews
+                    currentPlace.reviews.forEach(function(review) { review.user_id = review.user.id; });
+                    let transformedCurrentPlaceReviews = normalizerService.normalize({ data: currentPlace.reviews }, {
```

Comment:
this can be like `getReviewSchema` method to return
```
{
                        created_at: '',
                        description: '',
                        dislikes: 0,
                        like: 'NONE',
                        likes: 0,
                        user_id: 0
 }
```